joplin/api/schema.py

```diff
@@ -378,7 +378,59 @@ class Meta:
         interfaces = [graphene.Node]
 
 
-class DepartmentPageTopServiceNode(DjangoObjectType):
+class DepartmentPageTopPageNode(DjangoObjectType):
```

Comment:
I can't think of a good strategy to avoid this copypasta at the moment, but it'd be cool if we could!

Maybe we can find a place to put down ideas/set aside some time to research?

Reply:
I think a pure issue to research / review / refactor for our naming conventions, architecture here would great investment!